### DISCUSSION TITLE
Connect to database using Django

### DISCUSSION THREAD

**Discussion by SwastikGorai** [appwrite/appwrite#5669] [Q&A] [answered]
I came across appwrite recently and I found out that they provide database service. I'd like to know how can i use the database and connect it to a django project?

**Comment by joeyouss** [comment, marked as answer]
Hi
These links will help you:

https://appwrite.io/docs/getting-started-for-server

https://appwrite.io/docs/server/databases?sdk=python-default#databasesCreateDocument
